src/middlewared/middlewared/etc_files/local/smb4_share.conf

```diff
@@ -9,21 +9,15 @@
 
         def get_db_config():
             db = {}
+            db['globals'] = middleware.call_sync('sharing.smb._get_global_params', None)
```

Comment:
When we compose service from multiple files (which I think we'll need to do now) we don't import methods whose names begin with `_`.  Our new agreement is to use normal method names and just make them private.

Reply:
Okay. I will switch methods to not begin with an underscore.